Turkey/Test.cs

```diff
@@ -18,8 +18,10 @@ public class TestDescriptor
         public string Type { get; set; }
         public bool Cleanup { get; set; }
         public double TimeoutMultiplier { get; set; } = 1.0;
-        public List<string> IgnoredRIDs { get; set; } = new();
-        public List<string> SkipWhen { get; set; } = new();
+#pragma warning disable CA1051 // Do not declare visible instance fields
+        public List<string> IgnoredRIDs = new();
```

Comment:
Add a `{get; set;}` so these become properties instead of instance fields. Then you can omit the warning supressions.